GitHubCard/index.js

```diff
@@ -46,7 +66,67 @@ const followersArray = [];
 
 */
 
-/* List of LS Instructors Github username's: 
+function cardCreator(userObj) {
+  const newDiv = document.createElement('div')
+  newDiv.classList.add('card')
+
+  const userImg = document.createElement('img') // Ask how to create `src`
+  userImg.src = userObj.avatar_url
+  newDiv.appendChild(userImg)
+
+  const newDiv2 = document.createElement('div')
+  newDiv2.classList.add('card-info')
+  newDiv.appendChild(newDiv2)
+
+  const h3 = document.createElement('h3')
+  h3.classList.add('name')
+  h3.textContent = userObj.name
+  newDiv2.appendChild(h3)
+
+  const userName = document.createElement('p')
+  userName.classList.add('username')
+  userName.textContent = userObj.login
+  newDiv2.appendChild(userName)
+
+  const userLocation = document.createElement('p')
+  userLocation.textContent = userObj.location
+  newDiv2.appendChild(userLocation)
+
+  const userProfile = document.createElement('p')
+  userProfile.textContent = 'Profile: '
+  newDiv2.appendChild(userProfile)
+
+  const userURL = document.createElement('a') // ask how to create `href`
+  userURL.textContent = userObj.html_url
+  userURL.href = userObj.html_url
+  userProfile.appendChild(userURL) // make sure you're appending it to the right parent
+
+  const followers = document.createElement('p')
+  followers.textContent = userObj.followers
+  newDiv2.appendChild(followers)
+
+  const following = document.createElement('p')
+  following.textContent = userObj.following
+  newDiv2.appendChild(following)
+
+  const userBio = document.createElement('p')
```

Comment:
Your component function looks fine, but can you think of a way to make it dryer?